packages/cli/test/integration.js

```diff
@@ -1813,31 +1813,6 @@ test('remove the wildcard alias', async t => {
 });
 */
 
-test('ensure username in list is right', async t => {
```

Comment:
Why do we not need these test anymore?

Reply:
I decided to remove this test because it's no longer necessary now that there are unit tests that cover this. It will also start failing once I remove the username field from the table, so I figured it was better not to deal with it.